### DISCUSSION TITLE
Package request

### DISCUSSION THREAD

**Discussion by rrs45** [nanovms/ops#1493] [Q&A]
Is it possible to create Ops packages for following opensource tools. These already have docker images, wondering if nanos can run them.

- Appsmith
- Open Search
- Sonarqube

**Comment by eyberg** [comment]
I think both appsmith && open-search are totally doable

we already made one for elastic and that's essentially the same as opensearch https://repo.ops.city/v2/packages/nanovms/elasticsearch/8.2.0/x86_64/show - so should be relatively straight-forward to cut a new build of opensearch

I took a quick peek at appsmith as well - we can definitely create a pkg out of that although that has 5 services would be worth creating a new compose file (see https://docs.ops.city/ops/compose) for that since it has 5? services (mongo, redis, backend on jetty, rts, frontend) - mongo, redis we already have pkgs for so we'd just need to build pkgs for the other 3

sonarqube - i haven't personally used and haven't checked it out yet

can feel free to open tkts to track these

**Reply by rrs45** [reply]
Can ops compose be used for any cloud provider? How can service discovery be accomplished in that case?

**Reply by eyberg** [reply]
there's no cloud today that would support 'ops compose' - that is really a dev-only thing right now

**Comment by eyberg** [comment]
first stab @ opensearch - https://repo.ops.city/v2/packages/eyberg/opensearch/2.8.0/x86_64/show - it could definitely use some better default configuration
